src/components/AddItem/AddItem.js

```diff
@@ -4,37 +4,82 @@ import NavBar from '../NavBar/NavBar';
 
 const AddItem = () => {
   const [groceryItem, setGroceryItem] = useState('');
-  const [purchaseDate, setPurchaseDate] = useState(null);
   const [howSoon, setHowSoon] = useState(null);
-  const listData = firestore.collection('groceryItems');
+  // const listData = firestore.collection('groceryItems'); commented out due to not wanting to direct to the 'groceryItems' collection but use the entire database instead
 
   const itemInputChange = (e) => {
     setGroceryItem(e.target.value);
   };
 
-  const dateInputChange = (e) => {
-    setPurchaseDate(e.target.value);
-  };
   const radioInputChange = (e) => {
     setHowSoon(parseInt(e.target.value));
     console.log(howSoon);
   };
 
+  const cleanUserInput = (item) => {
+    // Step 1: Sanitizing and trying out the regular expression symbols and converting to lowercase
+    return item.replace(/[\W_]+/g, '').toLowerCase();
+  };
+
   const handleSubmitClick = async (e) => {
     e.preventDefault();
-    await listData.add({
-      // writing a new document to firestore with the select values/fields
+
+    const existingItem = await duplicateItem(groceryItem); // Step 2: I'm trying to add a new function as a addon that will check for dupe items with an alert
```

Comment:
I'd suggest moving this comment to the line above.
```suggestion
    // Step 2: I'm trying to add a new function as a addon that will check for dupe items with an alert
    const existingItem = await duplicateItem(groceryItem); 
```

**Why:** By doing this you could get two benefits:
1. Consistency: All the comments in the codebase are treated in the same way.
2. Readability: Having this in separate lines makes it easer to see for future readers.